test/rebar3_edoc_extensions_SUITE.erl

```diff
@@ -1,31 +0,0 @@
--module(rebar3_edoc_extensions_SUITE).
```

Comment:
Please revert those changes - this is a part of base skeleton and and these tests will expand with adding fake application to check rebar3 plugin behavior.

Reply:
I understand it's part of the base skeleton, but nothing is tested currently. We can add it back when there is an actual testsuite.

Reply:
I will start working on it soon.